src/common/config/constants.ts

```diff
@@ -285,21 +285,14 @@ export const OPERATOR_NAME_CNV = 'cnv';
 export const OPERATOR_NAME_LSO = 'lso';
 export const OPERATOR_NAME_ODF = 'odf';
 export const OPERATOR_NAME_LVM = 'lvm';
-export const OPERATOR_NAME_CVO = 'cvo';
-export const OPERATOR_NAME_CONSOLE = 'console';
-
-export type OperatorName =
-  | typeof OPERATOR_NAME_CNV
-  | typeof OPERATOR_NAME_LSO
-  | typeof OPERATOR_NAME_ODF
-  | typeof OPERATOR_NAME_LVM
-  | typeof OPERATOR_NAME_CVO
-  | typeof OPERATOR_NAME_CONSOLE;
-
-export const operatorLabels = (t: TFunction): { [key in OperatorName]: string } => ({
-  [OPERATOR_NAME_CONSOLE]: t('ai:OpenShift Console'),
-  [OPERATOR_NAME_CVO]: t('ai:OpenShift Cluster Version Operator'),
-  [OPERATOR_NAME_LSO]: t('ai:OpenShift Local Storage'),
+
+const OperatorNames = [OPERATOR_NAME_CNV, OPERATOR_NAME_LSO, OPERATOR_NAME_ODF, OPERATOR_NAME_LVM];
+export const ExposedOperatorNames = [OPERATOR_NAME_CNV, OPERATOR_NAME_ODF, OPERATOR_NAME_LVM];
```

Comment:
If at any time we expose LSO to the users (or stop using `OPERATOR_NAME_LSO` in the code), we can remove the distinction between operators and exposed operators.